FlowCrypt/Common UI/AttachmentManager.swift

```diff
@@ -49,13 +50,19 @@ extension AttachmentManager: AttachmentManagerType {
             .sinkFuture(
```

Comment:
@tomholub I think we should use `async/await` here instead of Combine for code consistency across project, and for better threads handling with `@MainActor`, can be done in another issue

Reply:
Agree - I've asked @ekievsky to do it as part of #1031

Reply:
Will also file another issue to look over the remaining ones after that, if any